examples/eks-cluster-mgmt/charts/kro/rgs/pod-identity/pod-identity.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: kro.run/v1alpha1
+kind: ResourceGroup
+metadata:
+  name: podidentity.kro.run
+  annotations:
+    argocd.argoproj.io/sync-wave: "-1"
+spec:
+  schema:
+    apiVersion: v1alpha1
+    kind: PodIdentity    
+    spec:
+      name: string
+      role:
+        policies: "[]string"
+  resources: # how to publish a field in the RG claim e.g. vpcID
+  - id: podrole
+    template:
+      apiVersion: iam.services.k8s.aws/v1alpha1
+      kind: Role
+      metadata:
+        name: ${schema.spec.name}-pod-role
+      spec:
+        name: ${schema.spec.name}-pod-role
+        policies: ${schema.spec.role.policies}
+        assumeRolePolicyDocument: |
+          {
+            "Version": "2012-10-17",
+            "Statement": [
+              {
+                "Effect": "Allow",
+                "Principal": {
+                  "Service": "pods.eks.amazonaws.com"
+                },
+                "Action": [
+                  "sts:TagSession",
+                  "sts:AssumeRole"
+                ]
+              }
+            ]
+          }
+
```

Comment:
Is the PodIdentityAssociation resource missing?

Reply:
Ye the role and pod association was not added yet i need to finalise it